client/src/controllers/TagController.ts

```diff
@@ -18,24 +19,88 @@ declare global {
  */
 export class TagController extends Controller {
   static values = {
+    delay: { type: Number, default: 300 },
```

Comment:
Let's default this to zero (`0`) and then apply the value of `200` at the widget level, using `300` is a bit too slow.

I can add this when merging though + adjust the unit test.